src/app/services/auth.service.ts

```diff
@@ -91,6 +92,14 @@ export class AuthService {
     return this.http.post<any>(`${this.baseUrl}/confirmemail`, confirmEmail);
   }
 
+  forgotPassword(dto: {usernameOrEmail: string}): Observable<any> {
+    const createNewPasswordDto  = {
+      usernameOrEmail: dto.usernameOrEmail,
+      clientUri: `${environment.web.protocol}://${environment.web.baseUrl}/forgotPassword`,
```

Comment:
bei ClientUri muss der Ziellink rein, der hinter dem Button in der Mail verlinkt ist, die man durch den API Aufruf bekommt. Die Komponente zum Eintragen des neuen Passwortes gibt es nicht, solltest du zumindest als neue Komponente anlegen, z.B. unter onboarding, entsprechend müsste dann der Link heißen:

clientUri: `${environment.web.protocol}://${environment.web.baseUrl}/onboarding/forgotPassword`,

![image](https://user-images.githubusercontent.com/77295035/109410538-bc3df180-799b-11eb-800a-0415918613cd.png)